### PR TITLE
Metrics app capstone

### PR DESCRIPTION
**Pull request by ahmed-al-farouq**
What I did:
- [ ] initialized the react app
- [ ] fetched continents and countries
- [ ] created navbar, home, and details page
- [ ] added styling
- [ ] created testing for fetching and components
- [ ] created readme file
- [ ] deployed the site
- [ ] created a loo video link below:
https://www.loom.com/share/257ed309326048609b16bcc013432231

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,61 @@
-# Getting Started with Create React App
+# Metrics Webapp
+
```
Comment (rloterh) [inline README.md:2]: The video presentation showcasing your project is **incomplete**. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The video should:
- [ ]  contain a **clear and objective description** of the project in professional English. The topic and purpose of the project are made clear.
- [ ]  contain a demo of the project, **highlighting all the project features**.
- [ ]  contain a **highlight about a piece of the code** the student finds interesting or is proud of making.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Omar, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to complete this project but you are almost there! 🚦  🚦  

## Good Points 👍 
- App is correctly deployed
- Redux is effectively used to manage state data
- Test cases are passing
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ahmed-al-farouq) [reply README.md:2]: Thanks for the feedback sir.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ahmed-al-farouq,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**